threat-model/readme.md

```diff
@@ -213,6 +213,28 @@ TODO
 #### 1.9.2 Mitigations
 TODO
 
+### 1.10 TEE
+
+TEEs are hardware supported environments for computations that isolate associated processes from other processed on the host machine. TEEs are intended to provide confidentiality, integrity, and authenticity for computations against a, possibly malicious, operator of the host machine.
```

Comment:
```suggestion
TEEs are hardware supported environments for computations that isolate associated processes from other processed on the host machine. TEEs are intended to provide confidentiality, integrity, and authenticity for computations against a possibly malicious operator of the host machine or - in a shared hosting environment - other users of the same host.
```

Reply:
good catch on the other users. Do we want to call them users or is there a better term? I also like "colocated" for processes assigned to the same host in cloud environments.